core/src/main/java/io/aiven/klaw/service/EnvsClustersTenantsControllerService.java

```diff
@@ -632,17 +632,9 @@ public ApiResponse addNewEnv(EnvModel newEnv) throws KlawException {
 
     newEnv.setTenantId(tenantId);
 
-    if (newEnv.getClusterId() == null) {
-      return ApiResponse.builder().result("Please select a cluster.").build();
-    }
```

Comment:
I don't see a replacement check for this are we sure we want to remove the cluster check?

Reply:
We have a notnull validation check on clusterid

@NotNull private Integer clusterId;